env.dist

```diff
@@ -5,97 +5,63 @@
 WEB_CONCURRENCY=1
 
 # development or production
-export NODE_ENV="development"
+NODE_ENV="development"
 
 # default thimble port is 3500
-export PORT=3500
+PORT=3500
 
 # hostname for this server according to your browser
 # if they don't match then Persona will not work
-export APP_HOSTNAME="http://localhost:3500"
+APP_HOSTNAME="http://localhost:3500"
 
 # secret used for signed cookies
-export SESSION_SECRET="dummy secret value"
+SESSION_SECRET="dummy secret value"
 
 # Optional google analytics settings - if used, they will inject
 # the google analytics JS snippet with your information filled in.
-export GA_ACCOUNT=
-export GA_DOMAIN=
+GA_ACCOUNT=
+GA_DOMAIN=
 
 # Is this server running behind an SSL-enabled load-balancer?
-export FORCE_SSL=false
+FORCE_SSL=false
 
 # i18n Language Support
-export L10N_SUPPORTED_LANGUAGES="[ "*" ]"
-export L10N_LOCALE_SRC="locales"
-export L10N_LOCALE_DEST="dist/locales"
+L10N_SUPPORTED_LANGUAGES="[ "*" ]"
```

Comment:
I'm sure this is right, but the way the `"`s are embedded here is odd.

Reply:
yeah it's weird but the only reason we do this is because we can rely on habitat which parses it for us. Regular env parsing won't parse it correctly, we would need to manually `JSON.parse` it.